classes/03class/exercises/c03-compose01/alanlima/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.3"
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: tellnoone
+  web:
+    build:
+      context: ../../../compose/artifacts/c05-compose01
+      dockerfile: Dockerfile
+    ports:
+      - "8100:80"
+    volumes: 
+      - type: bind
+        source: ../../../compose/artifacts/c05-compose01/src
+        target: /var/www/html
+    environment: 
+      DB_USER: root
+      DB_PASS: tellnoone
+      DB_HOST: db
```

Comment:
Good usage of the internal Docker network, using `db` as the host without exposing ports